std/concurrency.d

```diff
@@ -1504,7 +1415,9 @@ private:
     auto cond = fs.newCondition(mtx);
 
     size_t received, sent;
-    auto waiter = new Fiber({receive(cond, received);}), notifier = new Fiber({send(cond, sent);});
+    auto waiter = new Fiber({ receive(cond, received); }), notifier = new Fiber({
+        send(cond, sent);
+    });
     waiter.call();
```

Comment:
This looks a bit weird as newlines are used only for the second `Fiber` expression